tgui/packages/tgui/interfaces/ExaminePanel.jsx

```diff
@@ -50,165 +58,253 @@ export const ExaminePanel = (props) => {
     character_ad,
     headshot,
     headshot_nsfw,
+    art_ref,
+    art_ref_nsfw,
   } = data;
+
+  const handlePageChange = (page, newPage) => {
+    setPage(newPage);
+  };
+
   return (
     <Window
       title={character_name + "'s Examine Panel"}
       width={900}
       height={670}
     >
       <Window.Content>
-        <Stack fill>
-          <Stack.Item width="30%">
-            {!headshot ? (
-              <Section fill title="Character Preview">
-                <ByondUi
-                  height="100%"
-                  width="100%"
-                  className="ExaminePanel__map"
-                  params={{
-                    id: assigned_map,
-                    type: 'map',
-                  }}
-                />
-              </Section>
-            ) : (
-              <>
-                <Section height="310px" title="Character Preview">
-                  <ByondUi
-                    height="260px"
-                    width="100%"
-                    className="ExaminePanel__map"
-                    params={{
-                      id: assigned_map,
-                      type: 'map',
-                    }}
-                  />
-                </Section>
-                <Section height="310px" title="Headshot">
-                  <img
-                    src={
-                      tabIndex === 2
-                        ? resolveAsset(headshot_nsfw)
-                        : resolveAsset(headshot)
-                    }
-                    height="250px"
-                    width="250px"
-                  />
-                </Section>
-              </>
+        {page === 'main' ? (
```

Comment:
using ternary to choose the page isn't that great, maybe add functions within ExaminePanel that return the different pages? Then you could add error handling if it's not one of the valid choices as using strings is easy to mess up.